themes/default/content/docs/using-pulumi/testing/unit.md

```diff
@@ -223,14 +223,16 @@ We use the built-in `go test` command, so no need to install anything.
 
 We use [NUnit](https://nunit.org/) test framework to define and run the tests, [Moq](https://github.com/moq/moq4) for mocks, and [FluentAssertions](https://github.com/fluentassertions/fluentassertions) for assertions.
 
-Install the corresponding NuGet packages to your program:
+Install the corresponding NuGet and Pulumi packages to your program:
```

Comment:
These are all considered NuGet packages. ( remove 'and Pulumi' )

Reply:
```suggestion
Install the corresponding NuGet packages to your program:
```